src/components/CategoryFilter.js

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const CategoryFilter = ({ filterHandler }) => {
+  const categories = ['All', 'Action', 'Biography', 'History', 'Horror', 'Kids', 'Learning', 'Sci-Fi'];
+  return (
+    <form>
+      <select onChange={e => filterHandler(e.target.value)}>
+        { categories.map(category => (
+          <option key={category} value={category}>{category}</option>)) }
+      </select>
+    </form>
+  );
```

Comment:
* [ ] you should merge `All` select option with available categories instead of adding it to the array. ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/react_reducx/bookstore/milestone4/AllReturnsAllCategories.png) from Pathwright), add `All`, then loop through the array.